source/adminguide/host_and_storage_tags.rst

```diff
@@ -0,0 +1,92 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information#
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+   http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+
+
+Host and Storage Tags
+======================
+
+Host tags and storage tags, despite the name, are not related to resource tags (section :ref:`resource-tags`) and are a feature to direct resource allocation, such as which host the VM will be deployed on or which storage the volume will be created on. There are several reasons for using tags (such as directing the volume to better quality storage, according to the offering).
```

Comment:
@JoaoJandre the only thing i see different in this case is the braces "()"
maybe try
```suggestion
Host tags and storage tags, despite the name, are not related to resource tags, explained in section :ref:`resource-tags` and are a feature to direct resource allocation, such as which host the VM will be deployed on or which storage the volume will be created on. There are several reasons for using tags (such as directing the volume to better quality storage, according to the offering).
```

Reply:
I've made a commit removing the parenthesis, if this doesn't work, I'll remove the reference

Reply:
I've removed the reference, I have no idea what was the issue :shrug:

Reply:
I think I found it, all other anchors have a blank line following (before the title they adhere to) i.e.
```
.. _changing-root-password:

Changing the Root Password
~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Let's see how it goes now

Reply:
It finally worked, nice catch @DaanHoogland :)